media/js/firefox/os/partner_data.js

```diff
@@ -10,112 +10,97 @@ if (typeof Mozilla === 'undefined') {
 (function() {
     'use strict';
 
-    // create namespace
+  // create namespace
```

Comment:
OK , I need to turn off my eslint formatter on save. I'll fix this tomorrow.
Le 27 avr. 2016 10:04 PM, "Alex Gibson" <notifications@github.com> a écrit :

> In media/js/firefox/os/partner_data.js
> <https://github.com/mozilla/bedrock/pull/4085#discussion_r61326190>:
>
> > @@ -10,112 +10,97 @@ if (typeof Mozilla === 'undefined') {
> >  (function() {
> >      'use strict';
> >
> > -    // create namespace
> > +  // create namespace
>
> Likewise the comments here should match indentation to the code they
> relate to.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/bedrock/pull/4085/files/9d708f11f7ac7f93e1b1083c4c8c19b062c811b6#r61326190>
>